pkg/crypto/tinkcrypto/key_wrapper.go

```diff
@@ -489,11 +489,13 @@ func (t *Crypto) deriveESWithOKPKey(apu, apv []byte, recPubKey *cryptoapi.Public
 		base64.RawURLEncoding.Encode(apu, ephemeralPubKey)
 	}
 
-	kek, err := cryptoutil.Derive25519KEK([]byte(wrappingAlg), apu, apv, ephemeralPrivChacha, recPubKeyChacha)
+	z, err := cryptoutil.DeriveECDHX25519(ephemeralPrivChacha, recPubKeyChacha)
```

Comment:
What about the P curve version?

Reply:
this is in `deriveESWithOKPKey` which ECDH-ES using X25519 key
`cryptoutil.Derive25519KEK` was renamed to `cryptoutil.DeriveECDHX25519`  and now returns Z only. the KDF is done manually below now.

Reply:
the P curve for EC is plain ECDH-ES (it uses uses go-jose directly, see [above](https://github.com/hyperledger/aries-framework-go/pull/2767/files/45126548a79b1df9a6f3e1826a76229844cf372c#diff-572f9fe3f634c134a40c37a1d19ff6398faa6daeb5d8463ad797841560aa5952R400))